git-cliff/src/args.rs

```diff
@@ -346,9 +346,12 @@ impl TypedValueParser for RemoteValueParser {
 	) -> Result<Self::Value, clap::Error> {
 		let inner = clap::builder::StringValueParser::new();
 		let value = inner.parse_ref(cmd, arg, value)?;
-		let parts = value.split('/').rev().collect::<Vec<&str>>();
-		if let (Some(owner), Some(repo)) = (parts.get(1), parts.first()) {
-			Ok(RemoteValue(Remote::new(*owner, *repo)))
+		let parts = value.rsplit_once('/');
```

Comment:
One idea is to parse it as URL then split it (just to be safe):

```suggestion
		let mut value = inner.parse_ref(cmd, arg, value)?;
		if let Ok(url) = Url::parse(&value) {
			value = url.path().trim_start_matches('/').to_string();
		}
		let parts = value.rsplit_once('/');
```

(you need to add `url` crate to the workspace dependencies)